web/ui/module/lezer-promql/src/promql.grammar

```diff
@@ -231,7 +231,21 @@ MatchOp {
 }
 
 LabelMatcher {
-  LabelName MatchOp StringLiteral
+  UnquotedLabelMatcher |
+  QuotedLabelMatcher |
+  MetricNameLabelMatcher
```

Comment:
I think we could simplify the grammar like that: 

```txt
LabelMatchers {
  "{" ((UnquotedLabelMatcher | QuotedLabelMatcher | QuotedLabelName)("," (UnquotedLabelMatcher | QuotedLabelMatcher | QuotedLabelName))* ","?)? "}"
}
```

That would simplify the numbers of node in the tree. Specially regarding `MetricNameLabelMatcher` that doesn't bring a value from my point of view.

Reply:
Yeah, `MetricNameLabelMatcher` doesn't add any value I added that to make it more readable. But It can be simplified in this way.

Reply:
Yes I'm aware things will need to change in other files specially in the autocomplete part.
But It makes sense to change that.
`LabelMatcher` in its original shape is a combination of two strings and an operator.

Here that's not the case anymore and so the term `LabelMatcher` doesn't make any sense and so need to disappear.

Besides you already replace `LabelMatcher` by  `UnquotedLabelMatcher` so I'm pretty sure it won't change that much to remove it from the grammar.

Reply:
Yes, that's a good point. I'll replace the Label matcher. It makes more sense.

Reply:
I've simplified the grammar according to your suggestion. Sorry it took so long :))